### PR TITLE
Implement ESLint

### PR DESCRIPTION
**Pull request by ZoicanDenisUi**
Implementing ESLint for the Bounce project

### CODE REVIEW

**Comment by sergiubologa** [comment]
Please rebase so it can be reviewed.

File: Src/Helpers/domInteractions.ts [hunk 1]
```diff
@@ -1,48 +1,51 @@
...
-    parentDiv.removeChild(circleDom)
+export function removeCircleFromDom (circleDom:HTMLDivElement) {
+  parentDiv.removeChild(circleDom)
 }
 
-export function updateCircleRadiusInDom(circleDom:HTMLDivElement,newRadius:number){
-    circleDom.style.width = newRadius+"px"
-    circleDom.style.height = newRadius+"px"
+export function updateCircleRadiusInDom (circleDom:HTMLDivElement, newRadius:number) {
+  // eslint-disable-next-line no-param-reassign
```
Comment (sergiubologa) [inline Src/Helpers/domInteractions.ts]: Disabling rules is not ideal. Seems like this rule can be configured to allow this use case. See details here: https://eslint.org/docs/rules/no-param-reassign#props

File: package.json [hunk 2]
```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "live-server bin",
-    "compile": "concurrently --kill-others \"tsc -w\" \"live-server bin\""
+    "compile": "concurrently --kill-others \"tsc -w\" \"live-server bin\"",
+    "lint": "npx eslint .\\Src\\**\\*.ts",
+    "lintAndFix": "npx eslint .\\Src\\**\\*.ts --fix"
```
Comment (sergiubologa) [inline package.json]: why is npx needed here?

**Review by sergiubologa — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (ZoicanDenisUi) [reply package.json]: Deleted npx.

**Review by sergiubologa — APPROVED** [review]

**Pull request merged**